qlty-cli/src/initializer/templates/source_default.toml

```diff
@@ -1,3 +1,3 @@
 [[source]]
 name = "default"
-default = true
\ No newline at end of file
+default = true
```

Comment:
The error was the missing newline at end of this file, is that right?

Reply:
Yup, it was rendering `default = true[[source]]`